src/tomography.jl

```diff
@@ -0,0 +1,244 @@
+using Shortcuts, SCS, QIP, Convex
+
+function tomo_gate_set(nbrQubits, nbrPulses; pulse_type="Clifford", prep_meas = 1)
+#tomo_gate_set Returns a set of state preparation or readout unitary gates.
+
+#pulse_type: prepared states/meas. axes
+#prep_meas: 1 for prep., 2 for meas. pulses
+
+#Basic Pauli operators
+    X = [[0 1]; [1 0]];
+    Y = [[0 -1im]; [1im 0]];
+    Z = [[1 0]; [0 -1]];
+I = eye(2);
```

Comment:
In fact, how about we just use the Pauli matrices defined in Cliffords.jl? Just add a `using Cliffords` at the top and replace `I` with `Id`.